src/osd/OSD.cc

```diff
@@ -2512,6 +2513,25 @@ void OSD::create_logger()
 
   PerfCountersBuilder osd_plb(cct, "osd", l_osd_first, l_osd_last);
 
+  // Latency axis configuration for op histograms, values are in nanoseconds
+  PerfHistogramCommon::axis_config_d op_hist_x_axis_config{
+    "Latency (usec)",
+    PerfHistogramCommon::SCALE_LOG2, ///< Latency in logarithmic scale
+    0,                               ///< Start at 0
+    100000,                          ///< Quantization unit is 100usec
```

Comment:
Although the quantization is fixed, the axis scale is logarithmic here so that there's much higher granularity for lower values and much lower for the big ones.

I took a brief look at HdrHistogram and it does something very similar, just configured in different way. I think it won't be a good fit here though since they [don't support atomic/concurrent histograms](https://github.com/HdrHistogram/HdrHistogram_c#hdrhistogram). Maybe something other would be a better fit, do you have any experience in this area?

Also I'm pretty sure most of implementations would do histograms over one value, I try to gather it over two values simultaneously which may help much more fine-grained analysis.

I also remember Milosz Tanski was talking in ML about Count Min Sketch which we could look into.